Hunted Mobile/Hunted Mobile/Hunted Mobile/ViewModel/MapViewModel.cs

```diff
@@ -651,8 +658,11 @@ public class MapViewModel : BaseViewModel {
             }
 
             // If current user has role as Police
-            if(mapModel.PlayingUser is Police && mapModel.Thiefs.Count > 0) {
-                mapViewService.AddClosestThiefPin(GetClosestThief());
+            if(mapModel.PlayingUser is Police) {
+                mapViewService.AddPoliceStationPin(gameModel.PoliceStationLocation);
```

Comment:
Waarom wil je het politiebureau niet tonen van boefen? Want een boef weet dan toch ook niet waar die naartoe moet lopen als die gevangen is?

Reply:
Volgens mij wilden we juist dat de boeven het politiebureau niet zagen om gameplay redenen maar dat kijken we morgen dan wel even